platform/src/default.rs

```diff
@@ -10,11 +10,14 @@ pub const VENDOR_ID: u32 = 0;
 pub const VENDOR_SKU: u32 = 0;
 
 // Generated with openssl using the command openssl req -new -newkey ec -pkeyopt ec_paramgen_curve:prime256v1 -x509 -nodes -days 365 -out cert.pem -keyout cert.pem
-// Converted to der format using the command openssl x509 -in cert.pem -outform DER -out cert.der
+// Pasted key into a new file key_256.pem
+// Converted cert to der format using the command openssl x509 -in cert.pem -outform DER -out cert.der
+// Converted key to der format using the command openssl ec -in key_256.pem -outform DER -out key_256.der
```

Comment:
nit: Maybe a shell script in `test_data` for re-generating all of the test data would be useful. Then we can update these comments to just point to the shell script.

Reply:
A good portion of generating the test data was manual work (cut-pasting the private key into a new file, running the haskell script to format, and then removing quotations manually). I will look into a way to automate all of the generation via commands and then we can make the shell script.

Reply:
Ok. If it ends up being a lot of work, we can also do it in a follow-up PR

Reply:
Actually, I figured out a way to do it, I just had to split up generating the pkey and the cert.